prov/rxm/src/rxm_fabric.c

```diff
@@ -126,7 +126,41 @@ static int rxm_fabric_init_offload_coll(struct rxm_fabric *fabric)
 	 * silimar to rxm_fabric_init_util_coll, except that the offload
 	 * provider is discovered by feature instead of name.
 	 */
+	struct fi_info *hints, *offload_coll_info;
+	struct fid_fabric *offload_coll_fabric;
+	int ret;
+
+	hints = fi_allocinfo();
+	if (!hints)
+		return -FI_ENOMEM;
+
+	hints->fabric_attr->prov_name = strdup(OFI_OFFLOAD_PREFIX "sharp"); // XXX to be fixed
```

Comment:
Honestly, I'm okay discovering by name for the first submission.  It's easy and works for our purposes.  We can update later.